### DISCUSSION TITLE
Google login not working after publishing to Vercel

### DISCUSSION THREAD

**Discussion by Bobenheim** [vercel/community#1167] [Help] [answered]
Hi,
I've created a TikTik clone following a YT tutorial. On my local host the google login works fine, but after deployment on Vercel, when I click on the login button I just get a blank white window instead of the accounts choices.

Does anyone know what could be causing this?

here is the website: https://tiktikcloneapp.vercel.app/

github repository: [GitHub - Bobenheim/tiktikcloneapp](https://github.com/Bobenheim/tiktikcloneapp)

I have set the credentials for the new website, so this is not the issue. (https://i.stack.imgur.com/pBhB7.png)

This is the console error that I see when opening the website:
Failed to load resource: the server responded with a status of 403 ()
[GSI_LOGGER]: The given client ID is not found.

After clicking on the sign in button, nothing else shows up in the console.

**Comment by amyegan** [comment, marked as answer]
Hi @Bobenheim! A common cause of `403: Forbidden` responses is a missing or invalid token. Have you checked that you have all of the necessary production environment variables set for your project, and that they are not expired?

**Reply by Bobenheim** [reply]
Hi, unfortunately I've already checked that and all the variables are correct and none are expired.

**Reply by amyegan** [reply]
If you're sure you have the correct Client ID and have registered your project with Google, I'm not sure what else would cause a 403 response saying `The given client ID is not found` 🤔 

I tried finding other suggestions, but this was the only recent thing I could find: https://stackoverflow.com/questions/72955863/unable-to-get-clientid-from-googleoauthprovider

**Reply by Bobenheim** [reply]
Hi, 
I've tried creating a new ID and it works with that one ... I have honestly no idea why as the settings are exactly the same and I don't see that the previous one would be expired.
